### ISSUE TITLE
Play Button Overlay Lacks 3-1 Color Contrast

### ISSUE THREAD

**Issue by mshubat** [ableplayer/ableplayer#525]
Parts of an icon (with no text) do not have a contrast ratio of 3 to 1 against adjacent color(s). These icon parts are required for understanding.

Element: Play icon
Location: Located on the video in the video section

Color contrast ratio details:
Icon color: #EBE6E5
Adjacent background color: #DCD6D4
The contrast ratio is: 1.2:1

REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

![image](https://user-images.githubusercontent.com/10766136/173120469-ad6fc97b-735d-41dd-8d85-9caa5dcad78a.png)

**Comment by terrill** [comment]
@mshubat I'll accept this change since it's included in the bundle with all your other fixes. However, I wonder if there's a better solution. As a large element with no transparency, it obstructs the content in the background.  A couple of possible solutions: 

1. Change from a solid black background to a semi-transparent black background. Since we can't control for the colors in the background image, we could test with a white background and find the optimum level of transparency that would results in a 3:1 contrast ratio with a semi-transparent black overlay on a white background.  Or... 
2.  Remove the big play button altogether.  It's redundant since there's already a play button on the controller. However, in the 2020 skin the play button isn't the first element, so it's nice to have an earlier alternative. 

Thoughts?

**Comment by mshubat** [comment]
Hey @terrill, ya I see your point.

I think the play button is kind of nice to have: bigger touch/click point for the user as well.

It may be a little tricky to keep the 3:1 contrast ratio with a semi-transparent background, but we could give it a shot. Maybe a future enhancement that could be added? Or we continue on this ticket.

**Comment by terrill** [comment]
@mshubat let's continue on this ticket so we have the full context of our discussion. 

Here's an SVG play button to play with as a starting point, sitting on a background that's both dark and light. Tweak the opacity property of the button to test different values: 
http://terrillthompson.com/tests/svg/contrast.html

**Comment by mshubat** [comment]
@terrill ok thanks, that's a great starting point.

I will need to continue on ADA critical items for my team for now, but should be able to get back to this ticket shortly.

**Comment by mshubat** [comment]
@terrill I just got a chance to take a look at this again. I have opened a PR.

After playing around with different opacity levels I found an opacity of 0.75 was sufficient to achieve a 5.63 contrast ratio in the worst case scenario of pure black behind the white play icon, and pure white behind the black overlay, as well as provide sufficient transparency to see what is behind the icon.

![Screenshot 2022-08-08 141913](https://user-images.githubusercontent.com/10766136/183493301-189af3af-50ff-49ac-9c30-dfd561c7da45.png)

Here is a clip of how it looks/behaves.

![buttons-responsive](https://user-images.githubusercontent.com/10766136/183493741-9ff0688c-3764-492a-8870-32d13afe0da2.gif)

Let me know how this looks.

Thanks

**Comment by terrill** [comment]
@mshubat this looks great. Thanks for working on it!
